shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profilePreferences/presentation/editPersonalInfo/EditProfileScreenState.kt

```diff
@@ -0,0 +1,15 @@
+package com.vml.tutorial.plantshop.profilePreferences.presentation.editPersonalInfo
+
+import com.vml.tutorial.plantshop.core.presentation.UiText
+import com.vml.tutorial.plantshop.profilePreferences.domain.User
+
+data class EditProfileScreenState(
+    val user: User? = null,
+    val firstName: String? = user?.firstName,
+    val lastName: String? = user?.lastName,
+    val phoneNumber: String? = user?.phoneNumber,
+    val showDatePickerDialog: Boolean = false,
+    val birthDate: String? = user?.birthDate.takeUnless { it.isNullOrEmpty() } ?: "DD/MM/YYYY",
```

Comment:
Can you move the "hardcoded" part into UI? :) or it would be better if we could have it in strings :p

Reply:
I refactored the code so it is no longer part of the UI anymore. I could move it into the companion object but I think it would be too much only for one constant

Reply:
I changed my mind and put it into a companion object for the sake of code readability